### ISSUE TITLE
delete admin/aapscp

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5065]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#5065|
|Type|enhancement|
|Created|2011-04-24 18:56:42Z|
|Branch|https://github.com/twisted/twisted/tree/delete-aapscp-5065|

I think this file is part of a Windows release process that is long since gone.

<details><summary>Searchable metadata</summary>

```
trac-id__5065 5065
type__enhancement enhancement
reporter__glyph glyph
priority__normal normal
milestone__ 
branch__branches_delete_aapscp_5065 branches/delete-aapscp-5065
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1303671402000000 1303671402000000
changetime__1305495105000000 1305495105000000
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

(In [31835]) Branching to 'delete-aapscp-5065'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [31836]) remove windows release tool that is no longer used, refs [#5065](https://github.com/twisted/twisted/issues/5065)

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_4"></a>|@jonathanj set owner to @thijstriemstra|
|-|-|

Looks good to merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [31904]) Merge delete-aapscp-5065: Removed tool that used to be part of the win32 release process and is no longer used.

Author: thijs
Reviewer: jonathanj
Fixes: [#5065](https://github.com/twisted/twisted/issues/5065)
